### PR TITLE
[15.0][IMP] project_close_forecast_date: some improvements

### PR DESCRIPTION
**Pull request by IriaAlonso**
Some improvement:
- adds masive funtion to change last_close_date
- forecast_date opcional hide
- domain wrong forecast_date < last_close_date
- translations

### CODE REVIEW

**Comment by IriaAlonso** [comment]
@dalonsod can you review this?

File: project_close_forecast_date/wizard/project_close_month.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import models, fields
+
+
+class ProjectCloseMonth(models.TransientModel):
+    _name="project.close.month.wizard"
+    _description="project.close.month.wizard"
+
+    project_ids = fields.Many2many("project.project")
+    last_close_date = fields.Date()
```
Comment (dalonsod) [inline project_close_forecast_date/wizard/project_close_month.py:12]: @ChristianSantamaria  a last close date could be unset, or should this field be required?

File: project_close_forecast_date/wizard/project_close_month.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    def close_month(self):
+        for record in self.project_ids.filtered(lambda x: x.last_close_date < self.last_close_date):
+            record.write({"last_close_date":self.last_close_date})
```
Comment (dalonsod) [inline project_close_forecast_date/wizard/project_close_month.py]: This could be achieved in a single statement, as `write()` can be `multi`:
```suggestion
        self.project_ids.filtered(lambda x: x.last_close_date < self.last_close_date).write({"last_close_date":self.last_close_date})
```

File: project_close_forecast_date/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_project_close_month_wizard_manager","access.project_close_month_wizard_manager","project_close_forecast_date.model_project_close_month_wizard","project.group_project_manager",1,1,1,1
```
Comment (dalonsod) [inline project_close_forecast_date/security/ir.model.access.csv:2]: This suggests that only Project Managers can execute this new action (or wizard launch will fire an error). In `project_profitability_fields` implementation `last_close_date` can be edited by a Project User. Why this change in such permissions? Should relax this, or require being Project Manager in general for every `last_close_date`?

**Review by dalonsod — CHANGES_REQUESTED** [review]
@IriaAlonso @ChristianSantamaria see comments

**Comment by dalonsod** [comment]
After last review, looks good! @ChristianSantamaria  could you install them in CrtTest environment?

@IriaAlonso could you fix certain typos in commit messages:
* [IMP] project_close_forecast_date: adds masive funtion to change last_close_date => [IMP] project_close_forecast_date: adds _massive function_ to change last_close_date
* [IMP] project_close_forecast_date: forecast_date opcional hide  => [IMP] project_close_forecast_date: _make purchase line_ forecast_date _optional_ hide 
*  [FIX] project_close_forecast_date: domain wrong forecast_date < last_close_date =>  [FIX] project_close_forecast_date: _added missing domain_ forecast_date < last_close_date

@IriaAlonso could you investigate adding security restrictions to `action_close_month`, so only Project Managers can access it?

**Comment by IriaAlonso** [comment]
@dalonsod done

**Comment by ChristianSantamaria** [comment]
@dalonsod done instaled in test enviroment

**Comment by ChristianSantamaria** [comment]
@dalonsod

**Comment by ChristianSantamaria** [comment]
@dalonsod done :+1: :+1:

**Comment by dalonsod** [comment]
I see...

imagen

It's not the better appearance, but it shows information enough :+1: 

One last thing... could we add an order by clause, by project last close date, for the project tree?

**Comment by ChristianSantamaria** [comment]
> I see...
> 
> imagen
> 
> It's not the better appearance, but it shows information enough 👍
> 
> One last thing... could we add an order by clause, by project last close date, for the project tree?

Done

**Review by dalonsod — APPROVED** [review]
